model/textparse/openmetricsparse.go

```diff
@@ -652,10 +652,8 @@ func (p *OpenMetricsParser) parseLVals(offsets []int, isExemplar bool) ([]int, e
 
 // isCreatedSeries returns true if the current series is a _created series.
 func (p *OpenMetricsParser) isCreatedSeries() bool {
-	var newLbs labels.Labels
-	p.Metric(&newLbs)
-	name := newLbs.Get(model.MetricNameLabel)
-	if typeRequiresCT(p.mtype) && strings.HasSuffix(name, "_created") {
+	metricName := p.series[p.offsets[0]-p.start : p.offsets[1]-p.start]
+	if typeRequiresCT(p.mtype) && len(metricName) >= 8 && string(metricName[len(metricName)-8:]) == "_created" {
```

Comment:
`len(metricName) >= 8` is neat ;p probably deserves a comment?

Reply:
im using the opposite in another place
i feel like its kinda implied but ill add